GDWeave/Loader/PackFileLoader.cs

```diff
@@ -49,6 +49,15 @@ public IEnumerable<Token> Modify(string path, IEnumerable<Token> tokens) {
                     yield return new Token(TokenType.ParenthesisClose);
                     yield return token;
 
+                    // mod.Manifest.Id.add_to_group("weave_mod")
+                    yield return new IdentifierToken($"{mod.Manifest.Id}");
+                    yield return new Token(TokenType.Period);
+                    yield return new IdentifierToken("add_to_group");
+                    yield return new Token(TokenType.ParenthesisOpen);
+                    yield return new ConstantToken(new StringVariant("weave_mod"));
```

Comment:
It's "GDWeave", not "Weave", so this should probably be named "gdweave_mod" instead

Reply:
bruh